src/main/java/org/wmn4j/io/musicxml/MusicXmlReaderDom.java

```diff
@@ -141,17 +142,46 @@ private void readScoreToBuilder(ScoreBuilder scoreBuilder, Document doc) {
 	 * ScoreBuilder.
 	 */
 	private void readScoreAttributesToBuilder(ScoreBuilder scoreBuilder, Document doc) {
-		// TODO: Extend this when the attributes of a score are increased.
-		final Node movementTitle = doc.getElementsByTagName(MusicXmlTags.SCORE_MOVEMENT_TITLE).item(0);
-		if (movementTitle != null) {
-			scoreBuilder.setAttribute(Score.Attribute.TITLE, movementTitle.getTextContent());
+
+		String scoreTitle = null;
+		Node workNode = doc.getElementsByTagName(MusicXmlTags.SCORE_WORK).item(0);
+		if (workNode != null) {
+			Optional<Node> titleNode = DocHelper.findChild(workNode, MusicXmlTags.SCORE_WORK_TITLE);
+			if (titleNode.isPresent()) {
+				scoreTitle = titleNode.get().getTextContent();
+			}
+		}
+
+		// Some notation programs might put the title in the movement-title
+		// element instead of the actual work title element. Use movement-title as the score title in that case.
```

Comment:
I guess there could be a separate test for this? Maybe not entirely necessary :stuck_out_tongue: Also, is there a possibility of a bug if score title has been left empty intentionally and movement-title isn't?

Reply:
I changed this so that the work title and movement title are handled separately and reorganized the commits.

Reply:
Cool :+1: